lib/App/Perlbrew/Util.pm

```diff
@@ -151,4 +152,21 @@ sub make_skaji_relocatable_perl_url {
     return undef;
 }
 
+sub prompt {
```

Comment:
Is it also worth checking `PERL_MM_USE_DEFAULT` too?

Reply:
_Change is evil, unless it's the kind of change that leaves things exactly as they were_